pylint/extensions/comparison_placement.py

```diff
@@ -0,0 +1,64 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+
+"""Checks the placement of comparisons"""
+
+
+from astroid import nodes
+
+from pylint.checkers import BaseChecker, utils
+from pylint.interfaces import IAstroidChecker
+
+REVERSED_COMPS = {"<": ">", "<=": ">=", ">": "<", ">=": "<="}
+COMPARISON_OPERATORS = frozenset(("==", "!=", "<", ">", "<=", ">="))
+
+
+class ComparisonPlacementChecker(BaseChecker):
```

Comment:
Is this a checker that is supposed to stay with one message forever, we could use the same name than the message if that's the case ?
```suggestion
class MisplacedComparisonConstantChecker(BaseChecker):
```